### PR TITLE
[WIP] Issue #32 Adding archaeologist script

### PR DESCRIPTION
**Pull request by hferentschik**
Addresses issue #32

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM node:7.5.0
 
 LABEL maintainer "Minishift Team"
-ENV REFRESHED_AT 2017-07-08
+ENV REFRESHED_AT 2017-09-21
```
Comment (budhrg) [inline Dockerfile]: @hferentschik What does this indicate? Today's date? then it should be 2017-08-21 i guess

Comment (hferentschik) [inline Dockerfile]: Yesterdays really, but you are right. The actual date does not matter here. It is just a "trick" to force in some cases a fresh build

**Comment by hferentschik** [comment]
Pushed update switching to 'archaeologist' as name

**Comment by hferentschik** [comment]
Merged (as first cut)

**Pull request merged**